src/distribution-integration/alpm-hook.sh

```diff
@@ -2,12 +2,13 @@
 
 set -euo pipefail
 
-# enumerate files that are managed by holo-files
-holo scan --short | grep '^file:' | cut -d: -f2- | sort > /tmp/files-managed-by-holo
-
-# changed files come from stdin (but without leading slash!); find files
-# managed by Holo among these
-sort | sed 's+^+/+' | comm -12 - /tmp/files-managed-by-holo | sed 's/^/file:/' | xargs -r holo apply
-
-# cleanup
-rm /tmp/files-managed-by-holo
+# ALPM calls this script from the root directory, with respect to the
+# `--root` flag.
+export HOLO_ROOT_DIR="."
```

Comment:
> Any non-empty `HOLO_ROOT_DIR` will force plugins into testing mode.

Is that actually a problem (will one of them do something silly during `scan`)?  The hook will only apply `files:` entities, and at this point `holo-files` has *no* testing mode.